File: 1041.py
p = input().split()
x, y = p

x = float(x)
y = float(y)

if x == 0:
    if y == 0:
        print('Origem')
    if y != 0:
        print('Eixo Y')
if y == 0:
    if x != 0:
        print('Eixo X')
if x > 0:
    if y > 0:
        print('Q1')
    if y < 0:
        print('Q4')
if x < 0:
    if y > 0:
        print('Q2')
    if y < 0:
        print('Q3')
        

File: C#/1041.cs
using System;
using System.Linq;
using System.Globalization;

namespace URI
{
    class Program
    {
        static void Main(string[] args)
        {
            CultureInfo.DefaultThreadCurrentCulture = new CultureInfo("en-US");
            decimal[] ponto = Console.ReadLine().Split().Select(decimal.Parse).ToArray();
            decimal x = ponto[0];
            decimal y = ponto[1];

            if(x >0 && y >0) { Console.WriteLine("Q1"); }
            else if(x <0 && y <0) { Console.WriteLine("Q3"); }
            else if(x>0 && y<0){ Console.WriteLine("Q4"); }
            else if(x<0 && y > 0) { Console.WriteLine("Q2"); }
            else if(x==0 && y!=0) { Console.WriteLine("Eixo Y"); }
            else if (y == 0 && x!=0) { Console.WriteLine("Eixo X"); }
            else { Console.WriteLine("Origem"); }


        }
    }
}